packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/linkedin/appatidp.md

```diff
@@ -1,19 +1,25 @@
-1. Go to [LinkedIn Developer page](https://developer.linkedin.com/) and register for a developer account if you haven't already done so.
+1. Sign up for a [LinkedIn Developer](https://www.linkedin.com/developers/) account if you haven't already done so.  
 
-2. Create a LinkedIn app using these [instructions](https://www.linkedin.com/developer/apps).
+2. Create an app in the [LinkedIn developer](https://www.linkedin.com/developers/apps/new) portal.
 
-3. Save the **Client ID** and **Client Secret** values (located on the **Auth** tab) so that you can add them to the Okta configuration in the next section.
+3. From your app, click on the **Auth** tab and make note of the **Client ID** and **Client Secret** values so that you can add them to the Okta configuration in the next section.
```

Comment:
```suggestion
3. From your app, click **Auth** and make note of the **Client ID** and **Client Secret** values so that you can add them to the Okta configuration in the next section.
```